build.sbt

```diff
@@ -30,5 +30,8 @@ lazy val root = (project in file("."))
       "jitpack" at "https://jitpack.io"
     ),
     reStart / mainClass := Some("latis.server.Latis3Server"),
-    scalacOptions -= "-Xfatal-warnings"
+    scalacOptions -= "-Xfatal-warnings",
+    scalacOptions += {
+      if (insideCI.value) "-Wconf:any:e" else "-Wconf:any:w"
```

Comment:
Where are these options defined? Google is not my friend today.

Reply:
Do you mean `-Wconf`? There's a blog post about it here: https://www.scala-lang.org/2021/01/12/configuring-and-suppressing-warnings.html